sonic-xcvrd/tests/test_xcvrd.py

```diff
@@ -409,7 +455,7 @@ def test_y_cable_toggle_mux_torA_no_port_instance(self):
 
     def test_y_cable_toggle_mux_torA_update_status_exception(self):
 
-        # Maybe I should instantiate a port_instance mock object, give it the toggle_..._tor_a method as an attribute or something?
+        #ASHWIN: TODO: Maybe I should instantiate a port_instance mock object, give it the toggle_..._tor_a method as an attribute or something?
```

Comment:
Remove name

Reply:
Those items are done, I'll remove all the offending comments in the upcoming commit.

Reply:
Done in latest commit.